src/main/kotlin/lotto/model/game/Lottos.kt

```diff
@@ -0,0 +1,56 @@
+package lotto.model.game
+
+import lotto.model.result.Coincidence
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+class Lottos(val lottos: List<Lotto>) {
+    constructor(count: Int) : this(createLottos(count))
+
+    fun getEarningRate(winningLotto: Lotto): BigDecimal {
+        val totalPrizeMoney = getTotalPrizeMoney(winningLotto)
+        val budgetMoney = (lottos.size * COST_PER_ONE_LOTTO).toBigDecimal()
+
+        return totalPrizeMoney.divide(budgetMoney, TWO_DECIMAL_PLACES, RoundingMode.FLOOR)
+    }
+
+    fun getCoincidenceCount(coincidence: Coincidence, winningLotto: WinningLotto): Int {
+        return lottos.count { coincidence == it.getResult(winningLotto) }
+    }
+
+    fun add(other: Lottos): Lottos {
```

Comment:
코틀린에서는 정해진 이름의 함수를 오버로딩함으로써 표준 수학 연산자를 오버로딩할 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/operator-overloading.html

```suggestion
    operator fun plus(other: Lottos): Lottos {
```